src/ophyd_async/core/device_save_loader.py

```diff
@@ -190,9 +193,61 @@ def set_signal_values(
 
 
 def load_device(device: Device, path: str):
-    """
-    Sets all PV's to a device using a previously saved configuration
+    """Plan which loads PVs from a yaml file into a device.
+
+    Parameters
+    ----------
+    device: Device
+        The device to load PVs into
+    path: str
+        Path of the yaml file to load
+
+    See Also
+    --------
+    :func:`ophyd_async.core.save_device`
     """
     values = load_from_yaml(path)
     signals_to_set = walk_rw_signals(device)
     yield from set_signal_values(signals_to_set, values)
+
+
+def save_device(
+    device: Device,
+    path: str,
+    sorter: Callable[[Dict[str, Any]], Sequence[Dict[str, Any]]] = lambda x: [x],
+    ignore: Optional[List[str]] = None,
+):
```

Comment:
It would be nice to promote the lambda into a named function so we can reference it in the docs

```suggestion
def all_at_once(values: Dict[str, Any]) -> Sequence[Dict[str, Any]]:
    """Sort all the values into a single phase so they are set all at once"""
    return [values]


def save_device(
    device: Device,
    path: str,
    sorter: Callable[[Dict[str, Any]], Sequence[Dict[str, Any]]] = all_at_once,
    ignore: Optional[List[str]] = None,
):
```